labelme/utils/measure_working_time.py

```diff
@@ -15,11 +15,7 @@ def __init__(self, crypto_mode = True):
         self.working_count = 0
         self.worker_name = ''
         self.init_write_worker_name = True
-        current_platform = platform.system()
-        self.split_char = '\\'
-        if current_platform == 'Linux':
-            self.split_char = '/'
-        self.name_file_path = sys.path[0] + self.split_char + 'worker_name.txt'
+        self.name_file_path = os.path.join(sys.path[0] + 'worker_name.txt')
```

Comment:
```suggestion
        self.name_file_path = os.path.join(sys.path[0], 'worker_name.txt')
```
`os.path.join()` 함수 용법은 `+`로 더하는것이 아니라 path 합성하고 싶은 인자들을 `,`로 구분하여 여러 개 넣는 것으로 알고있습니다~

Reply:
헛.. 수정 완료했습니다~!